packages/shared/src/components/post/PostEngagements.tsx

```diff
@@ -61,6 +62,9 @@ function PostEngagements({
 
   const onCommented = (comment: Comment, isNew?: boolean) => {
     if (isNew) {
+      postAnalyticsEvent(AnalyticsEvent.CommentPost, post, {
+        extra: { commentId: comment.id, origin: 'comment modal' },
```

Comment:
Similar to the `open comment modal` event, we kept the origin here to make the data consistent.

Reply:
Just to check this triggers and comments and reply to comments right?

Reply:
replies should have different event if I remember correctly

Reply:
@idoshamun I don't seem to find a different one in the analytics taxonomy.
Did we maybe miss it initially?

Reply:
Oh you're right! We use the same event `comment post` but the `commentId` field in extra should indicate the parent comment if it's a reply. @sshanzel please adjust

Reply:
Actually also the origin is not actually correct anymore, it can't be the modal as it no longer exists.
Should probably check with @idoshamun or Dave what's more appropriate. (And maybe ask to update the taxonomy sheet for this one)

Reply:
We don't use `origin` for the `comment post` in our analysis right now. Since it's kinda streamlines already as you mentioned it's safe to ignore

Reply:
Amazing. I will update the `commentId` to be coming from the parent comment's id.